src/Core/constants.js

```diff
@@ -2,9 +2,8 @@ const I18Y = {
   hi: 'Hello, my dear',
   qus: 'How are you?',
   ans: 'I am exhausted',
-  number: [0, 1, 2, 3, 4, 5, 6, 7, 8, 9],
   message: ['Lorem, ipsum dolor sit amet  ', ' praesentium tempore atque eveniet', 'architecto Placeat inventore officiis odit ipsa quibusdam', ' praesentium tempore atque eveniet', 'consectetur adipisicing elit. Officiis error'],
-  name: ['Sergei', 'Andrey', 'Natalya', 'Igor', 'Egor'],
 };
+const dialogsData = [{ id: 1, name: 'Dymuch' }, { id: 2, name: 'Andrey' }, { id: 3, name: 'Natalya' }, { id: 4, name: 'Igor' }, { id: 5, name: 'Egor' }];
```

Comment:
Don't forget to add the indentations or press the Shift + Alt + F.
Additionally, consider adding a basic eslint + prettier with Airbnb Style Guide configs to the project.

Reply:
I sometimes foget to press.I so sorry